source/material_model/diffusion_dislocation.cc

```diff
@@ -84,8 +84,8 @@ namespace aspect
                  && stress_iteration < stress_max_iteration_number)
             {
 
-              const std::pair<double, double> diff_edot_and_deriv = diffusion_creep.compute_strain_rate_and_derivative(stress_ii, pressure, temperature, diffusion_creep_parameters);
-              const std::pair<double, double> disl_edot_and_deriv = dislocation_creep.compute_strain_rate_and_derivative(stress_ii, pressure, temperature, dislocation_creep_parameters);
+              const std::pair<double, double> diff_edot_and_deriv = diffusion_creep.compute_strain_rate_and_derivative(stress_ii, pressure, temperature, 1., diffusion_creep_parameters);
```

Comment:
```suggestion
              const std::pair<double, double> diff_edot_and_deriv = diffusion_creep.compute_strain_rate_and_derivative(stress_ii, pressure, temperature, water_concentration, diffusion_creep_parameters);
```

Would this work since `1.` is the default value for the parameter?

Reply:
Same for places where `1.` is defined instead of `water_concentration`.